src/vs/workbench/parts/extensions/electron-browser/extensions.contribution.ts

```diff
@@ -217,6 +217,11 @@ Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Configuration)
 				type: 'boolean',
 				description: localize('extensionsShowRecommendationsOnlyOnDemand', "If set to true, recommendations will not be fetched or shown unless specifically requested by the user."),
 				default: false
+			},
+			'extensions.closeExtensionDetailsOnViewChange': {
+				type: 'boolean',
+				description: localize('extensionsCloseExtensionDetailsOnViewChange', "If set to true, extension information tabs will be automatically closed upon navigating away from the Extensions View."),
+				default: false
```

Comment:
Or even `extension editors`?

Reply:
As someone who was very recently an outsider, `extension editors` was pretty confusing at first because it sounds like a place I'd go to if I wanted to edit an extension. But if that's the standard nomenclature I can change it. I think Ramya's `editors with extension details` works, if `tabs` is a bad word.